### DISCUSSION TITLE
IOS--- (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64

### DISCUSSION THREAD

**Discussion by aravindrv** [FreeRDP/FreeRDP#7035] [Q&A]
HI

I successfully created Xcode project file from the guidelines of the below url's (https://github.com/FreeRDP/FreeRDP/blob/master/docs/README.ios) .But I am unable to run the Xcode project for building the frame work.kindly help me on this issue to solve.

my Xcode version is Ver 11.6
MY Mac OS version Ver 10.15.6 (Catalina )
Mac book pro (13-inch,2020)

my error on Xcode is "Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture arm64"

kindly help me t0 resolve this issue.

Note::I tried to set build setting " ENABLE_BITCODE=NO" for debug mode.But same error showing.

**Comment by akallabeth** [comment]
iOS is currently lacking a maintainer.
I can only talk about MacOS and there you just run
1. `cmake -GXcode .` 
2. `cmake --build .`

and you´re done. should be the same for iOS, but if some additional settings are required you might:

1.  find the required options at https://cmake.org/cmake/help/latest/
1. the CMake toolchain file might be out of date, so check if you can find one that is more recent

if you manage to fix your issues we´d be really happy about a pull request/some update on the wiki :)
